src/locales/en/skjema.ts

```diff
@@ -78,7 +78,8 @@ export const skjema = {
         paabegynt: {
             soknad: {
                 flertall: "started applications",
-                informasjon: "Please note that the started applications will be deleted after {{DAYS_BEFORE_DELETION}} days.",
+                informasjon:
+                    "Please note that the started applications will be deleted after {{DAYS_BEFORE_DELETION}} days.",
```

Comment:
Generelt bør sånne verdier helst hete "count", for da får man litt ekstra automagi